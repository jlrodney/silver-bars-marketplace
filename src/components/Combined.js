import React from 'react';

const Combined = (props) => (
  <div>
  <p>{props.quantity} kg for Â£{(props.price)}</p>
  </div>
);

export default Combined;
