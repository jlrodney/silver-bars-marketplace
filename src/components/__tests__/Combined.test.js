import React from 'react';
import { shallow } from 'enzyme';
import Combined from '../Combined';

const order = [ 22, 230 ]
describe('Expense item', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Combined
      key={order[0]}
      quantity={order[1]}
      price={order[0]}
    />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly', () => {
    const wrapper = shallow(<Combined
      key={order[0]}
      quantity={order[1]}
      price={order[0]}
    />);
    expect(wrapper.find('p').text()).toEqual('230 kg for Â£0.22');
  });
});
