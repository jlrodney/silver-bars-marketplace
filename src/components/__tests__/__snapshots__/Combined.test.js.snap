// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Combined Order item should render correctly 1`] = `
<div>
  <p>
    230
     kg for Â£
    22
  </p>
</div>
`;
